ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1602,6 +1602,20 @@ literals:
 - valve_status:
   - OPEN
   - CLOSED
+
+- low_smoke_alarm: # for aspirating type smoke detectors
```

Comment:
Please update this field to be `low_smoke_intensity_alarm` (it is more descriptive that way and ensures the field can be re-used by other teams). You will need to add `intensity` to the subfields.yaml in the `descriptor` section. You can define it as `intensity: "The relative magnitude or strength of something  (I.e., high intensity indicates something is strong or forceful)."`

Reply:
Yes it is better what you suggest. Thank you, I'll update this and the relevant smoke alarms I have done.